index.js

```diff
@@ -0,0 +1,52 @@
+document.getElementById("text").value = "";
```

Comment:
store this function call in a variable so you don't have to call the same function again and again. `document.getElementById("text")`